crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -67,33 +67,38 @@ impl Sphere {
     }
 }
 
-/// An unbounded plane in 3D space. It forms a separating surface through the origin,
-/// stretching infinitely far
+/// A bounded plane in 3D space. It forms a surface starting from the origin with a defined height and width.
 #[derive(Clone, Copy, Debug, PartialEq)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
 pub struct Plane3d {
```

Comment:
I would be concerned about changing the semantics of a public type like this, but this just changes it back to what it was before right?

Reply:
The original `Plane` in `bevy_render` was bounded (but had no configurable normal and was always facing +Y), but the `Plane3d` primitive was always unbounded.

If we want to split it into a finite plane and and an infinite plane though, then I think this semantic change is kinda unavoidable, unless we instead name this something like `FinitePlane3d` or `BoundedPlane3d`, but that might be less user-friendly

Reply:
Agreed. We just need to highlight it really clearly in the migration notes.

Reply:
I think it there is no code change required by this one then, correct?

Reply:
Correct